### ISSUE TITLE
Unsupported URL: tvzavr.ru

### ISSUE THREAD

**Issue by SlazE888** [ytdl-org/youtube-dl#22935]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running youtube-dl version **2019.10.29**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single video: https://www.tvzavr.ru/film/rok1/
- Single video: https://www.tvzavr.ru/film/odin-den/
- Single video: https://www.tvzavr.ru/film/11-hd/

## Description

All video examples above plays fine in Google Chrome browser.

**Comment by SlazE888** [comment]
@remitamine you are wrong. No need account. Try to watch in Google Chrome browser and you will see.

**Comment by remitamine** [comment]
the first URL shows a buttor with this text:
> Смотреть по подписке за $1

the second plays a trailer.
the third shows this text:
> К сожалению, этот фильм пока не появился у нас на сайте
Сообщить мне при появлении

**Comment by SlazE888** [comment]
> the first URL shows a buttor with this text:
> 
> > Смотреть по подписке за $1
> 
> the second plays a trailer.
> the third shows this text:
> 
> > К сожалению, этот фильм пока не появился у нас на сайте
> 
> and a button with this text:
> 
> > Сообщить мне при появлении
@remitamine i got it. You can watched these movies without account only if you connect to this site through proxy from Russia! Thank you for pointing me.
